app/controllers/users_controller.rb

```diff
@@ -8,12 +8,21 @@ def create
     @user = User.create(user_params)
     if @user.save
       session[:user_id] = @user.id
-      redirect_to posts_url
+      redirect_to user_path(@user.email)
```

Comment:
Cool! Lets remember to build the functionality so a user's wall can be accessed with their user_id too.